### ISSUE TITLE
Crash with OOM error on large image

### ISSUE THREAD

**Issue by anonyuu** [jacklul/e621-Batch-Reverse-Search#18]
I have one 2.1 MB image that causes this tool to consistently crash with:
```
[12/168] Searching '1575180225943.jpg':
 Trying reverse search #1 (e621.net/iqdb_queries)...PHP Fatal error:  Allowed memory size of 134217728 bytes exhausted (tried to allocate 24576 bytes) in phar:///***/e621-Batch-Reverse-Search/build/e621BRS.phar/app.php on line 1375

Fatal error: Allowed memory size of 134217728 bytes exhausted (tried to allocate 24576 bytes) in phar:///***/e621-Batch-Reverse-Search/build/e621BRS.phar/app.php on line 1375
Press ENTER key to continue...
```
It'd be nice if the tool would at the least gracefully handled this case and ignored the image with a message that it was too large. It took me three runs before I figured out what the cause was.

**Comment by jacklul** [comment]
You can't really gracefully handle out of memory error.

I have memory limit set to '1024M' for people using binaries from 'runtime' folder.
You must be using Linux as the limit is 128M for you - you might wanna change 'memory_limit' in your `php.ini` for your system

It should be somewhere: `/etc/php.ini /usr/bin/php5/bin/php.ini /etc/php/php.ini /etc/php5/apache2/php.ini`

**Comment by jacklul** [comment]
Should be fixed in 1.7.3.

**Comment by anonyuu** [comment]
Changing php.ini worked. Thanks for the help.
